authappliance/lib/appliance.py

```diff
@@ -381,13 +381,19 @@ def __init__(self, client="/etc/freeradius/clients.conf"):
         Clients are always kept persistent on the file system
         :param client: clients.conf file.
         '''
-        # clients
-        self.config_file = client
-        self.ccp = ClientConfParser(infile=client)
-        self.config_path = os.path.dirname(client)
-        self.dir_enabled = self.config_path + "/sites-enabled"
-        self.dir_available = self.config_path + "/sites-available"
-        
+        # check if the clients.conf file exists and is readable
+        client3 = "/etc/freeradius/3.0/clients.conf"
+        if os.access(client, os.F_OK | os.R_OK | os.W_OK):
+            self.config_file = client
+        elif os.access(client3, os.F_OK | os.R_OK | os.W_OK):
+            self.config_file = client3
+        else:
+            raise FileNotFoundError()
```

Comment:
I think in previous version of the appliance the FreeRADIUS menu was simply hidden when the config file did not exist.
Is it still the same behaviour with the FileNotFoundError?

Otherwise the appliance tool would not work anymore, if the freeradius was removed or the file was renamed.

Reply:
This was the behavior before, the exception showed that there is no RADIUS configuration.